tools/test-static-code

```diff
@@ -122,7 +122,7 @@ else
     # all normal files in test/ and tools/ which are either:
     #   - filename is *.py
     #   - executable matching ^#!/usr/bin/python3
-    PYFILES="$(find test/ tools/ -type f '(' -name '*.py' -o -executable -execdir grep -q '^#!/usr/bin/python3' '{}' ';' ')'  -print)"
+    PYFILES="$(find . -type f '(' -name '*.py' -o -executable -execdir grep -q '^#!/usr/bin/python3' '{}' ';' ')'  -print)"
```

Comment:
This currently lives in the "various: pycodestyle changes" commit, but is meant to be in the "test-static-code: check all python files" commit. Please either move it, or squash the two commits together.

Reply:
done, thanks